downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,422 @@
+
+## Update Ballerina
+
+Update your current Ballerina installation directly to 2201.8.0 using the [Ballerina Update Tool](/learn/cli-documentation/update-tool/) as follows.
+
+1. Run `bal update` to get the latest version of the Update Tool.
+2. Run `bal dist update` to update to this latest distribution.
+
+## Install Ballerina
+
+If you have not installed Ballerina, download the [installers](/downloads/#swanlake) to install.
+
+>**Info:** From this release onwards, you can verify Ballerina artifacts using the Cosign CLI and Rekor APIs. For more information, see [Verify Ballerina artifacts](/downloads/verify-ballerina-artifacts).
+
+## Backward-incompatible changes
+
+- A type-checking bug that resulted in incorrect subtype relationships between records with optional fields and open records has been fixed.
+
+    ```ballerina
+    import ballerina/io;
+
+    type Person record {
+        string name;
+    };
+
+    type Employee record {
+        string name;
+        int id?;
+    };
+
+    public function main() {
+        Person person = {name: "May"};
+        Employee employee = person; // Compilation error now.
+
+        io:println(person is Employee); // Prints `false` now.
+    }
+    ```
+
+- Analysis in the `init` method of an `isolated` object has been updated to disallow invalid transferring in/out of values that violated the isolated root invariant.
+
+    ```ballerina
+    type Node record {|
+        readonly string path;
+        map<string[]>? app = ();
+    |};
+
+    isolated class Class {
+        private Node node;
+        private Node[] arr = [];
+
+        function init(Node node) {
+            self.node = node.clone();
+
+            self.arr.push(node); // Compilation error now.
+            self.arr[0] = node; // Compilation error now.
+        }
+    }
+    ```
+
+- Mutable objects have been disallowed in annotations.
+
+    ```ballerina
+    type Validator object {
+        function validate(anydata data) returns error?;
+    };
+
+    type AnnotationData record {|
+        string name;
+        Validator validator;
+    |};
+
+    annotation AnnotationData config on type; // Compilation error now.
+    ```
+    
+- Fixed a bug that allowed using field access with a map of `xml`. 
+
+    ```ballerina
+    map<xml> m = {a: xml `foo`};
+    xml x = check m.a; // Compilation error now.
+    ```
+
+- Ballerina interoperability implementation may have an impact with the Java 17 support due to any incompatible changes. For example, Java 17 has some restrictions on using Java reflections with internal Java packages. For more information, see the Java 17 release notes.
+
+- A bug that permitted uninitialized variables to evade detection when utilizing the `on fail` clause has been fixed.
+
+  ```ballerina
```

Comment:
Seems the code block is not properly formatted.

Reply:
Fixed.